mantidproject/mantid
# Added GDA saving to ISIS Powder Gem

Added new parameter `save_gda` to the Gem object that, when set will save out .gda files when focusing workspaces, this will be set by default when `texture_mode` is enabled. Since `gem_algs.py` was growing cluttered, created a new file `gem_output.py` and created the new save method there, as well as moving the older save methods over from `gem_algs.py`. edited the C++ parameter names of `SaveGDA` to match similar algorithms since it was still currently unused to prevent breaking changes once it was used in future.

Added Unit test for the ability to save .gda files.
Moved ISIS Powder tests to a subdirectory.


**To test:**
Copy the GEM_test folder from `\\olympic\babylon5\Public\Sam Jenkins` to your computer, then edit the `Gem_config_example.yaml` so that the paths within are complete. then run the following script.

```
from isis_powder import Gem

config_file_path = r"Path\To\Gem_config_example.yaml"
Gem_example = Gem(config_file=config_file_path,user_name="test",texture_mode=True)
Gem_example.create_vanadium(first_cycle_run_no="85105")
Gem_example.focus(input_mode="Individual",run_number="85106")
```
Then follow the instructions in the Issue for using Maud to check that the files match up.

Fixes #23153, #22502

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
